load_testing/scenarios.py

```diff
@@ -36,3 +37,26 @@ def test(url):
         test('/data/Discussion/{disc_id}/')
         test('/data/Discussion/{disc_id}/idea_links')
         test('/api/v1/discussion/{disc_id}/posts?{posts}&order=reverse_chronological&view=default')
+
+    def notification_subscriptions(self):
```

Comment:
From what I know, have to run this test against an actual `uswgi` server, not pserve, in order to get the full benefits of multiprocessing. Install instructions [here](https://uwsgi-docs.readthedocs.io/en/latest/Install.html). On mac, it's `brew install uwsgi`. Then start the service in the assembl project like this:

```
uwsgi --die-on-term --ini local.ini --ini-paste-logged local.ini
```